GCM/App/register/RegisterModel.java

```diff
@@ -30,37 +31,50 @@ public boolean validpassword(String password) {
 		return true;
 	}
 
-	public boolean validConfirmPassword(String password,String confirmPassword) {
-		if(!password.equals(confirmPassword)) {
+	public boolean validConfirmPassword(String password, String confirmPassword) {
+		if (!password.equals(confirmPassword)) {
 			return false;
 		}
 		return true;
 	}
-	
+
 	public boolean validName(String name) {
-		if(name.length()>10||name.length()<3) {
+		if (name.length() > 10 || name.length() < 3) {
```

Comment:
Move the numbers to a class constant (`private final int MAX_USERNAME_LENGTH` or something like that)